.github/actions/perform-deployment/action.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Perform deployment.'
```

Comment:
You meant want to explain (with examples) what a deployment really is / what it represents (in readme or here).

People who won't check the code itself may be confused on what deployment really is in this case.

Reply:
Added the DEPLOYMENT.md readme file.